core/trino-main/src/main/java/io/trino/operator/Driver.java

```diff
@@ -609,7 +611,8 @@ private Throwable closeAndDestroyOperators(List<Operator> operators)
         return Optional.empty();
     }
 
-    private static Throwable addSuppressedException(Throwable inFlightException, Throwable newException, String message, Object... args)
+    @FormatMethod
+    private static Throwable addSuppressedException(Throwable inFlightException, Throwable newException, final @FormatString String message, Object... args)
```

Comment:
remove `final`

Reply:
Can't - Error Prone insists that it has to be final or effectively final even though it's the latter; probably a bug in Error Prone, but still

Reply:
Actually, it looks like it was some kind of false positive and it's gone now, not sure why (different version of ErrorProne in the IDEA plugin perhaps). Removed the `final`, thanks.